_inc/client/main.jsx

```diff
@@ -267,6 +268,19 @@ class Main extends React.Component {
 	}
 
 	render() {
+		if ( this.props.route.path === '/plans-prompt' ) {
+			return (
+				<div>
+					<div className="jp-lower">
+						<AdminNotices />
```

Comment:
Seems like we could use the nav here too?

Reply:
What kind of nav do you mean, @tyxla?

Reply:
![](https://cldup.com/AMQEhTXItC.png)

Maybe both of these navs would make sense on that screen? The top one is the main screen one, and the second one is the one that actually would display the "Plans" as active page.

But FWIW, we can also decide to not display a nav here intentionally. A good reason would be not to have distraction from the plans. I'm sure @jeffgolenski would know better than me what's the best way to go there.

Reply:
Also, if we decide to do it, this could also be done in another PR. Let's see what @jeffgolenski thinks about it.

Reply:
Yes, I think the intention here is not to let the users skip this step. Let's wait for @jeffgolenski's insight.